testing/TestLibraryCode.c

```diff
@@ -0,0 +1,62 @@
+#include "string.h"
+#include "stddef.h"
+#include "unity.h"
+#include "unity_fixture.h"
+
+TEST_GROUP(LibraryCode);
+
+TEST_SETUP(LibraryCode){
+
+}
+
+TEST_TEAR_DOWN(LibraryCode){
+
+}
+
+TEST(LibraryCode, StrcmpReturnsCorrectAnswer)
+{
+	//test with long string
```

Comment:
Please use complete sentences (punctuation, articles) and C-style comments (`/* ... */`).

`sed -i 's@//test \(.*\)$@/* Test \1. */@' TestLibraryCode.c`

Reply:
done